src/Components/Endpoints/src/Builder/RazorComponentApplicationAttribute.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.Components.Infrastructure;
+
+/// <summary>
+/// The definition of a Razor Components Application.
+/// </summary>
+/// <remarks>
+/// Typically the top level component (like the App component or the MainLayout component)
+/// for the application implements this interface.
```

Comment:
Which interface? Is this comment on the wrong class?

Reply:
Sorry, this is leftover (I converted the interface on to the attribute)